src/psfmachine/tpf.py

```diff
@@ -96,6 +96,15 @@ def __init__(
         self.time_corrector = time_corrector
         self.cartesian_knot_spacing = cartesian_knot_spacing
         self.bkg_subtracted = bkg_subtracted
+        # limiting mag values to consider contamination for kepler & tess
+        # hardcoded values to work with kepler
+        if self.nsources / self.npixels > 0.2:
+            if self.tpf_meta["mission"][0].lower() == "kepler":
+                self.cont_mag_limit = 17.5
+            elif self.tpf_meta["mission"][0].lower() == "tess":
+                self.cont_mag_limit = 17
```

Comment:
Will the source density of 0.2 sources per pixel work well for TESS? I am guessing it will.

Reply:
I'll do a quick test